core/src/test/scala/unit/kafka/server/RequestQuotaTest.scala

```diff
@@ -49,6 +47,7 @@ import org.apache.kafka.common.message.RenewDelegationTokenRequestData
 import org.apache.kafka.common.message.SaslAuthenticateRequestData
 import org.apache.kafka.common.message.SaslHandshakeRequestData
 import org.apache.kafka.common.message.SyncGroupRequestData
+import org.apache.kafka.common.message._
```

Comment:
If we add this "import all", we should remove all the other import statements

Reply:
Good point, let me quickly correct this.